spec/requests/claims_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe "Today's Claims" do
+  let(:json) { JSON.parse(response.body) }
+
+  it "responds successfully" do
+    get "/api/v1/claims/today"
+    expect(response.code).to eq "200"
+  end
+
+  it "responds with json" do
+    get "/api/v1/claims/today"
+    expect(response["Content-Type"]).to eq "application/json; charset=utf-8"
+  end
+
+  it "lists users bringing donuts today" do
+    user1 = create(:user, name: "Laura Mosher", github_login: "laura")
+    user2 = create(:user, name: "Chris Gaffney", github_login: "chris")
+    create(:claim, user: user1)
+    create(:claim, user: user2)
```

Comment:
You may want to explicitly create these claims for _today_ rather than counting on the factory to do so. The test (rather than the factory) should set the data that it needs.